ReactWindows/ReactNative.Shared/Reflection/EnumHelpers.cs

```diff
@@ -41,6 +39,30 @@ public static T Parse<T>(string value)
             return Parse<T>(value);
         }
 
+        private static Dictionary<string, object> EnumToDictionary(Type type)
+        {
+            var result = new Dictionary<string, object>();
+
+            var names = Enum.GetNames(type);
+            var values = Enum.GetValues(type);
+
+            for (int i = 0; i < values.Length; ++i)
+            {
+                string name = names[i];
+                object value = values.GetValue(i);
+
+                result.Add(Normalize(name), value);
+
+                var enumMemberAttribute = type.GetField(name).GetCustomAttributes(typeof(EnumMemberAttribute), true).SingleOrDefault();
+                if (enumMemberAttribute != null)
+                {
+                    result.Add(Normalize(((EnumMemberAttribute)enumMemberAttribute).Value), value);
```

Comment:
It works both ways. For example, 
`Newtonsoft.Json.JsonConvert.SerializeObject(TextDecorationLine.UnderlineLineThrough, new StringEnumConverter())`
will respect the EnumMemberAttribute (notice the StringEnumConverter, without this enum is serialized as int)